ColumbusStateWorkforceInnovation/java-3-model-view-controller-lab
# Fixing Intellij Project Settings

## Context ##
When students try to load this project in Intellij the project fails to open. This is because the .iml file has not be provided and the project settings are misconfigured (expected JDK, API level, JUnit dependency, etc). This PR corrects the project configuration such that a student can simply download the project and open it in Intellij without error.

## Changes ##
- [X] Updated the `.gitignore` file to remove `*.iml` files.
- [X] Recreated the `.idea` folder (most of this folder is checked in, excluding the `.idea/workspace.xml` file).
- [X] Added the project `.iml` file to version control.

## Validation ##
1. Check out the feature branch.
2. Open the project in Intellij.
3. Validate that the project opens successfully with the project structure.
4. Validate that JDK 8 has been set for the project.
5. Validate that the API level is Java 8.
6. Validate that the tests can be run successfully without further configuration.
